Integrations/Devo/Devo_description.md

```diff
@@ -0,0 +1,173 @@
+
+**Examples**
+
+*reader_endpoint*
+```
+https://apiv2-us.devo.com/search/query
+```
+
+*reader_oauth_token*
+```
+IAMANOAUTHTOKEN
+```
+
+*writer_relay*
+```
+us.elb.relay.logtrust.net
+```
+
+*writer_credentials*
+```json
+{
+  "key":"FAKEKEYHERE",
+  "crt":"FAKECERTHERE",
+  "chain":"FAKECHAINHERE"
+}
+```
+
+*fetch_incidents_filters*
+```json
+[
+  {"key":"engine","value":"simultaneous_login", "operator":"->"},
+  {"key":"extraData","value":"test@test.com","operator":"->"},
+  {"key":"extraData","value":"ANOMALOUS","operator":"->"}
+]
+```
+
+### Devo (Preview) Commands ###
+
+`devo-run-query`
+
+Runs a linq query and returns results from the specified timeframe.
+
+*inputs*
+
+| Argument       | Required | Type   | Description                                                                                                                                                        |
+|----------------|----------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| query          | TRUE     | string | A linq query that you wish to run against Devo                                                                                                                     |
+| from           | TRUE     | float  | Start datetime for specified Query. Expects a unix timestamp in seconds with milliseconds allowed after decimal points.                                            |
+| to             | FALSE    | float  | End datetime for specified Query. If no value is provided will default to now. Expects a unix timestamp in seconds with milliseconds allowed after decimal points |
+| writeToContext | FALSE    | string | String of `true` or `false` to pass on the results context.                                                                                                        |
+
+*outputs*
+
+| ContextPath       | Type          | Description                                                                                                                                         |
+|-------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Devo.QueryResults | list(objects) | Will return a list of python dictionary objects. Each object in the list will be a row from the query. If no rows are returned result will be null. |
+| Devo.QueryLink    | object        | Will return an JSON object with a single key called DevoTableLink and a URL link back to Devo for the results just queried.                         |
+
+*current caveats*
+- None
```

Comment:
_description file should only contain instructions on how to configure the integration in order to connect to Devo instance.
For each parameter means like you explained above.
Remove the documentation about the commands. For commands you need to create README.md file in this folder and generate documentation file and fill it with data.
Follow: https://github.com/demisto/content/blob/master/docs/integration_documentation/README.MD